### ISSUE TITLE
Support for Python 3.12

### ISSUE THREAD

**Issue by ZedongPeng** [rtonoue/pulp2mat#9]
It looks like the current `pulp2mat` only supports `Python >=3.8,<3.12`. Is there any plan to make it support `Python 3.12`?
```
python -m pip install pulp2mat
ERROR: Ignored the following versions that require a different python version: 0.1.1 Requires-Python >=3.8,<3.12; 0.1.2 Requires-Python >=3.8,<3.12; 0.1.3 Requires-Python >=3.8,<3.12
ERROR: Could not find a version that satisfies the requirement pulp2mat (from versions: none)
ERROR: No matching distribution found for pulp2mat
```

**Comment by rtonoue** [comment]
@ZedongPeng, 
Thank you for your feedback.
v0.1.5 is now released and it supports python 3.12 and later.

**Comment by ZedongPeng** [comment]
Thank you so much @rtonoue .
